zcash_client_sqlite/src/wallet/db.rs

```diff
@@ -135,10 +137,6 @@ CREATE TABLE ephemeral_addresses (
 // libsqlite3-sys requires at least version 3.14.0.
 // "WITHOUT ROWID" tells SQLite to use a clustered index on the (composite) primary key.
 const_assert_eq!(GAP_LIMIT, 20);
-pub(super) const INDEX_EPHEMERAL_ADDRESSES_ADDRESS: &str = r#"
-CREATE INDEX ephemeral_addresses_address ON ephemeral_addresses (
-    address ASC
-)"#;
```

Comment:
For reference, the `ephemeral_addr_uniq` constraint has [essentially the same effect as a unique index](https://www.sqlite.org/forum/info/5e249867c3ed37ce).